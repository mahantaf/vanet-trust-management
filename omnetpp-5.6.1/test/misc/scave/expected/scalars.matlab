scalars.description="scalar desc"
scalars.File=[
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
"testfiles/scalars.sca";
]

scalars.Run=[
"scalars.sca:7-#35";
"scalars.sca:11-#34";
"scalars.sca:15-#1";
"scalars.sca:19-#2";
"scalars.sca:23-#43";
"scalars.sca:27-#6";
"scalars.sca:31-#7";
"scalars.sca:35-#8";
"scalars.sca:39-#9";
"scalars.sca:43-#10";
"scalars.sca:47-#11";
"scalars.sca:51-#21";
"scalars.sca:55-#22";
"scalars.sca:59-#203";
"scalars.sca:63-#204";
"scalars.sca:67-#25";
"scalars.sca:71-#106";
"scalars.sca:75-#107";
"scalars.sca:79-#28";
"scalars.sca:83-#29";
]

scalars.Module=[
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
"Net.subnet[0]";
]

scalars.frames_dropped=[
0;
NaN;
NaN;
0;
0;
0.002;
0;
0;
0.012;
0;
0.009;
0.014;
0.021;
0.022;
0.026;
0.051;
0.092;
0.144;
0.254;
0.434;
]

scalars.offered_load=[
30;
40;
10;
20;
50;
NaN;
70;
80;
90;
100;
110;
120;
130;
140;
150;
160;
170;
180;
190;
200;
]

scalars.throughput=[
0.3511;
0.4506;
0.0812;
0.2296;
0.5321;
0.5988;
NaN;
0.6981;
0.7347;
0.7646;
0.7891;
0.8092;
0.8257;
0.8391;
0.845;
0.8392;
0.8086;
0.7487;
0.6877;
0.5716;
]

