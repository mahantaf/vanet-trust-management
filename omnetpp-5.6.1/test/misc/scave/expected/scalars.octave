# name: scalars
# type: struct
# length: 7
# name: description
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 1
# length: 11
scalar desc
# name: File
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 20
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# length: 21
testfiles/scalars.sca
# name: Run
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 20
# length: 17
scalars.sca:7-#35
# length: 18
scalars.sca:11-#34
# length: 17
scalars.sca:15-#1
# length: 17
scalars.sca:19-#2
# length: 18
scalars.sca:23-#43
# length: 17
scalars.sca:27-#6
# length: 17
scalars.sca:31-#7
# length: 17
scalars.sca:35-#8
# length: 17
scalars.sca:39-#9
# length: 18
scalars.sca:43-#10
# length: 18
scalars.sca:47-#11
# length: 18
scalars.sca:51-#21
# length: 18
scalars.sca:55-#22
# length: 19
scalars.sca:59-#203
# length: 19
scalars.sca:63-#204
# length: 18
scalars.sca:67-#25
# length: 19
scalars.sca:71-#106
# length: 19
scalars.sca:75-#107
# length: 18
scalars.sca:79-#28
# length: 18
scalars.sca:83-#29
# name: Module
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: string
# elements: 20
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# length: 13
Net.subnet[0]
# name: frames_dropped
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0
NaN
NaN
0
0
0.002
0
0
0.012
0
0.009
0.014
0.021
0.022
0.026
0.051
0.092
0.144
0.254
0.434
# name: offered_load
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
30
40
10
20
50
NaN
70
80
90
100
110
120
130
140
150
160
170
180
190
200
# name: throughput
# type: cell
# rows: 1
# columns: 1
# name: <cell-element>
# type: matrix
# rows: 20
# columns: 1
0.3511
0.4506
0.0812
0.2296
0.5321
0.5988
NaN
0.6981
0.7347
0.7646
0.7891
0.8092
0.8257
0.8391
0.845
0.8392
0.8086
0.7487
0.6877
0.5716
