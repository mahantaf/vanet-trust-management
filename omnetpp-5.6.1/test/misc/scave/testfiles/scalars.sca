#
# Sample file for creating scatter plots.
#
# For the X axis, choose "Net.subnet[0] - throughput"
#
run 35
scalar "Net.subnet[0]"  "offered load"	30
scalar "Net.subnet[0]"  "throughput"	0.3511
scalar "Net.subnet[0]"  "frames dropped"	0
run 34
scalar "Net.subnet[0]"  "offered load"	40
scalar "Net.subnet[0]"  "throughput"	0.4506
#scalar "Net.subnet[0]"  "frames dropped"	0
run 1
scalar "Net.subnet[0]"  "offered load"	10
scalar "Net.subnet[0]"  "throughput"	0.0812
#scalar "Net.subnet[0]"  "frames dropped"	0
run 2
scalar "Net.subnet[0]"  "offered load"	20
scalar "Net.subnet[0]"  "throughput"	0.2296
scalar "Net.subnet[0]"  "frames dropped"	0
run 43
scalar "Net.subnet[0]"  "offered load"	50
scalar "Net.subnet[0]"  "throughput"	0.5321
scalar "Net.subnet[0]"  "frames dropped"	0
run 6
#scalar "Net.subnet[0]"  "offered load"	60
scalar "Net.subnet[0]"  "throughput"	0.5988
scalar "Net.subnet[0]"  "frames dropped"	0.002
run 7
scalar "Net.subnet[0]"  "offered load"	70
#scalar "Net.subnet[0]"  "throughput"	0.6534
scalar "Net.subnet[0]"  "frames dropped"	0
run 8
scalar "Net.subnet[0]"  "offered load"	80
scalar "Net.subnet[0]"  "throughput"	0.6981
scalar "Net.subnet[0]"  "frames dropped"	0
run 9
scalar "Net.subnet[0]"  "offered load"	90
scalar "Net.subnet[0]"  "throughput"	0.7347
scalar "Net.subnet[0]"  "frames dropped"	0.012
run 10
scalar "Net.subnet[0]"  "offered load"	100
scalar "Net.subnet[0]"  "throughput"	0.7646
scalar "Net.subnet[0]"  "frames dropped"	0
run 11
scalar "Net.subnet[0]"  "offered load"	110
scalar "Net.subnet[0]"  "throughput"	0.7891
scalar "Net.subnet[0]"  "frames dropped"	0.009
run 21
scalar "Net.subnet[0]"  "offered load"	120
scalar "Net.subnet[0]"  "throughput"	0.8092
scalar "Net.subnet[0]"  "frames dropped"	0.014
run 22
scalar "Net.subnet[0]"  "offered load"	130
scalar "Net.subnet[0]"  "throughput"	0.8257
scalar "Net.subnet[0]"  "frames dropped"	0.021
run 203
scalar "Net.subnet[0]"  "offered load"	140
scalar "Net.subnet[0]"  "throughput"	0.8391
scalar "Net.subnet[0]"  "frames dropped"	0.022
run 204
scalar "Net.subnet[0]"  "offered load"	150
scalar "Net.subnet[0]"  "throughput"	0.8450
scalar "Net.subnet[0]"  "frames dropped"	0.026
run 25
scalar "Net.subnet[0]"  "offered load"	160
scalar "Net.subnet[0]"  "throughput"	0.8392
scalar "Net.subnet[0]"  "frames dropped"	0.051
run 106
scalar "Net.subnet[0]"  "offered load"	170
scalar "Net.subnet[0]"  "throughput"	0.8086
scalar "Net.subnet[0]"  "frames dropped"	0.092
run 107
scalar "Net.subnet[0]"  "offered load"	180
scalar "Net.subnet[0]"  "throughput"	0.7487
scalar "Net.subnet[0]"  "frames dropped"	0.144
run 28
scalar "Net.subnet[0]"  "offered load"	190
scalar "Net.subnet[0]"  "throughput"	0.6877
scalar "Net.subnet[0]"  "frames dropped"	0.254
run 29
scalar "Net.subnet[0]"  "offered load"	200
scalar "Net.subnet[0]"  "throughput"	0.5716
scalar "Net.subnet[0]"  "frames dropped"	0.434



