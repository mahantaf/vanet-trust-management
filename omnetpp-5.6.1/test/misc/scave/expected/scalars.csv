File,Run,Module,frames dropped,offered load,throughput
testfiles/scalars.sca,scalars.sca:7-#35,Net.subnet[0],0,30,0.3511
testfiles/scalars.sca,scalars.sca:11-#34,Net.subnet[0],NaN,40,0.4506
testfiles/scalars.sca,scalars.sca:15-#1,Net.subnet[0],NaN,10,0.0812
testfiles/scalars.sca,scalars.sca:19-#2,Net.subnet[0],0,20,0.2296
testfiles/scalars.sca,scalars.sca:23-#43,Net.subnet[0],0,50,0.5321
testfiles/scalars.sca,scalars.sca:27-#6,Net.subnet[0],0.002,NaN,0.5988
testfiles/scalars.sca,scalars.sca:31-#7,Net.subnet[0],0,70,NaN
testfiles/scalars.sca,scalars.sca:35-#8,Net.subnet[0],0,80,0.6981
testfiles/scalars.sca,scalars.sca:39-#9,Net.subnet[0],0.012,90,0.7347
testfiles/scalars.sca,scalars.sca:43-#10,Net.subnet[0],0,100,0.7646
testfiles/scalars.sca,scalars.sca:47-#11,Net.subnet[0],0.009,110,0.7891
testfiles/scalars.sca,scalars.sca:51-#21,Net.subnet[0],0.014,120,0.8092
testfiles/scalars.sca,scalars.sca:55-#22,Net.subnet[0],0.021,130,0.8257
testfiles/scalars.sca,scalars.sca:59-#203,Net.subnet[0],0.022,140,0.8391
testfiles/scalars.sca,scalars.sca:63-#204,Net.subnet[0],0.026,150,0.845
testfiles/scalars.sca,scalars.sca:67-#25,Net.subnet[0],0.051,160,0.8392
testfiles/scalars.sca,scalars.sca:71-#106,Net.subnet[0],0.092,170,0.8086
testfiles/scalars.sca,scalars.sca:75-#107,Net.subnet[0],0.144,180,0.7487
testfiles/scalars.sca,scalars.sca:79-#28,Net.subnet[0],0.254,190,0.6877
testfiles/scalars.sca,scalars.sca:83-#29,Net.subnet[0],0.434,200,0.5716
